Assets/Scripts/Input/SwipeInteraction.cs

```diff
@@ -0,0 +1,76 @@
+using System.Collections;
+using System.Collections.Generic;
+using Unity.VisualScripting;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace Input
```

Comment:
Namespaces start with the project name and than the folder.

```suggestion
namespace UntitledCube.Input
```

Reply:
Changed it!